### ISSUE TITLE
Crash with OptiFabric v1.5.0 on multiplayer servers

### ISSUE THREAD

**Issue by maelstrom14** [Chocohead/OptiFabric#42]
But... with OptiFabric 1.4.10 everything fine!

```
---- Minecraft Crash Report ----
// Don't do that.

Time: 08.09.20 16:18
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraft.class_757.method_3188(class_757.java:987)
	at net.minecraft.class_757.method_3192(class_757.java:693)
	at net.minecraft.class_310.method_1523(class_310.java:1023)
	at net.minecraft.class_310.method_1514(class_310.java:656)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_757.method_3188(class_757.java:987)

-- Affected level --
Details:
	All players: 0 total; []
	Chunk stats: Client Chunk Cache: 121, 1
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 1 game time, 23700 day time
	Server brand: ~~ERROR~~ NullPointerException: null
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:617)
	at net.minecraft.class_310.method_1587(class_310.java:2183)
	at net.minecraft.class_310.method_1514(class_310.java:679)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)

-- System Details --
Details:
	Minecraft Version: 1.16.2
	Minecraft Version ID: 1.16.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_261, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2781415440 bytes (2652 MB) / 3221225472 bytes (3072 MB) up to 3221225472 bytes (3072 MB)
	CPUs: 4
	JVM Flags: 10 total; -Xmn128M -Xmx3072M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: fabric-loader-0.9.3+build.207-1.16.2
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 1050 Ti/PCIe/SSE2 GL version 4.6.0 NVIDIA 452.06, NVIDIA Corporation
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla (incompatible), file/Backpacks e100 (incompatible), file/UnobtrusiveRain.zip (incompatible), file/Clear Textures (incompatible)
	Current Language: Русский (Россия)
	CPU: 4x Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
	OptiFine Version: OptiFine_1.16.2_HD_U_G3
	OptiFine Build: 20200829-104659
	Render Distance Chunks: 5
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 452.06
	OpenGlRenderer: GeForce GTX 1050 Ti/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 4
```

**Comment by Chocohead** [comment]
What mods (if any) are you using alongside OptiFabric? I wouldn't have thought it would do this on its own.

**Comment by maelstrom14** [comment]
> What mods (if any) are you using alongside OptiFabric? I wouldn't have thought it would do this on its own.

OptiFine 1.16.2 HD U G3
OptiFabric 1.5.0

**Comment by Wolfieheart** [comment]
Can you check this with the latest version of Optifabric and see if it occurs?

**Comment by maelstrom14** [comment]
I checked, and crash only at servers with "Botfilter"

**Comment by Chocohead** [comment]
This looks like the issue which was fixed by Leymooo/BungeeCord@a2b64ed, if the servers are using an older version without the fix it might just crash with Optifine present in general.
